src/components/organisms/Room/Participant.tsx

```diff
@@ -26,7 +26,6 @@ type Track = VideoTracks[number] | AudioTracks[number];
 const Participant: React.FC<React.PropsWithChildren<ParticipantProps>> = ({
   participant,
   profileData,
-  profileDataId,
```

Comment:
Yes, it was never used in this component. However, the interface this component extends is used in other places as well. So, will not change the type

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/120 to investigate if this can be done safely for the other components, or if the type should be extracted somewhere more generic, etc